modules/account-lib/test/unit/coin/dot/transactionBuilder/addressInitializationBuilder.ts

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Do not comment out tests to skip them. We don't get any indication that there are skipped tests if it's done like this. Replace `describe` with `describe.skip()` instead

Reply:
Yes I talk with Alpay about the failing test and he fixed in https://github.com/BitGo/BitGoJS/pull/1841, Merge that one and I rebase this pr.